contracts/casino/src/casino.cpp

```diff
@@ -114,38 +121,41 @@ void casino::withdraw(name beneficiary_account, asset quantity) {
     if (account_balance > gstate.game_active_sessions_sum + gstate.game_profits_sum) {
```

Comment:
Potentially `gstate.game_profits_sum` can be negative, so in withdraw method we need to assume it as zero in case of that become negative. Because, if game has negative profit then game developer doesn't get any profit and here we must not consider that.